index.html

```diff
@@ -7,27 +7,30 @@
   <title>Rose's Portfolio - A full-stack Webdeveloper</title>
   <link rel="stylesheet" href="styles.css">
 </head>
-<body class="body-content">
+<body class="body-content" id="body">
   <header class="header">
       <div class="logo-section">
         <a href="/" class="logo">Rose</a>
       </div>
       <nav class="nav">
-        <div class="bars">
+        <div class="bars" onclick="OpenNavBarMenu()" id="hamburger">
```

Comment:
I think your tool bar still looks a bit different from the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=szEGBpxbDInIASLq-0)😉 Kindly take note of the difference between the screenshots below👍 Don't forget we are [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=szEGBpxbDInIASLq-0) in this milestone of the project😉

Your toolbar.
<img width="307" alt="hamb" src="https://user-images.githubusercontent.com/64643995/223183525-bf723712-f881-49c5-bcf4-edfe5b63a4bb.png">

design
![toolbar-mobile](https://user-images.githubusercontent.com/64643995/223183574-9bb33138-d280-483c-a539-8a8e466e5803.png)

Kindly take note of the spacing around the hamburger and make the necessary change😉